### PR TITLE
[RT-1404] Add a sentence on business manager and type change

### PR DESCRIPTION
**Pull request by huangalexpartoo**

### CODE REVIEW

File: rest_api/v2/components/index.yml [hunk 1]
```diff
@@ -451,12 +451,12 @@ schemas:
   Latitude:
     description: Latitude
     type: number
-    format: double
+    format: float
```
Comment (jonasbru) [inline rest_api/v2/components/index.yml:454]: why change it ?

File: rest_api/v2/components/index.yml [hunk 2]
```diff
@@ -605,11 +605,11 @@ schemas:
         $ref: '#/schemas/BusinessId'
       created:
         description: Timestamp (in seconds) of the business creation
-        type: number
+        type: integer
         example: 1409925979.5
```
Comment (jonasbru) [inline rest_api/v2/components/index.yml:609]: it doesn't look like an integer ^^ Would keep number

File: rest_api/v2/components/index.yml [hunk 3]
```diff
@@ -605,11 +605,11 @@ schemas:
...
       modified:
         description: Timestamp (in seconds) of the business last modification
-        type: number
+        type: integer
         example: 1561335111.681374
```
Comment (jonasbru) [inline rest_api/v2/components/index.yml:613]: it doesn't look like an integer ^^ Would keep number

File: rest_api/v2/components/index.yml [hunk 4]
```diff
@@ -1328,6 +1339,18 @@ schemas:
           type: integer
           description: If set, maximum value that this Custom Field will accept.
 
+  CustomFields.Metadata.Number:
```
Comment (jonasbru) [inline rest_api/v2/components/index.yml:1342]: You need to change it in line 1316 too

File: rest_api/v2/info/index.yml [hunk 5]
```diff
@@ -107,7 +107,7 @@ description: |
   | Category	 | not writable |                                                    |
 
   ## BUSINESS_MANAGER
-  `BUSINESS_MANAGER` role is maint for client business manager. It can manage several businesses.
+  `BUSINESS_MANAGER` role is maint for client business manager. It can manage several businesses, only if these businesses are in the same group.
```
Comment (jonasbru) [inline rest_api/v2/info/index.yml:110]: maint ? 

And I would say something about business without group.

File: rest_api/v2/paths/business/post_subscribe.yml [hunk 6]
```diff
@@ -13,7 +13,7 @@ requestBody:
         type: object
         properties:
           subscription_date:
-            type: number
+            type: integer
             description: Timestamp representing the date on which the subscription will start
             example: 1540628702
```
Comment (jonasbru) [inline rest_api/v2/paths/business/post_subscribe.yml:18]: timestamps can be floats

File: rest_api/v2/paths/business/post_unsubscribe.yml [hunk 7]
```diff
@@ -13,7 +13,7 @@ requestBody:
         type: object
         properties:
           canceling_date:
-            type: number
+            type: integer
```
Comment (jonasbru) [inline rest_api/v2/paths/business/post_unsubscribe.yml:16]: timestamps can be floats

File: rest_api/v2/info/index.yml [hunk 8]
```diff
@@ -40,7 +40,7 @@ description: |
     - `PROVIDER` role is meant for reseller admin user. It can manage organisations, users and businesses of a provider
     - `ORG_ADMIN` role is meant for client admin user. It can manage the user and businesses of its organisation
     - `GROUP_MANAGER` role is meant for client group manager. It can manage several businesses and users that belong to the group he managed
-    - `BUSINESS_MANAGER` role is meant for client business manager. It can manage several businesses
+    - `BUSINESS_MANAGER` role is meant for client business manager. It can manage several businesses, only if these businesses are in the same group.
```
Comment (jonasbru) [inline rest_api/v2/info/index.yml:43]: And while you're at it, I think we can be a bit better than "role is meant for client business manager" ^^ 

Same for others :)

File: rest_api/v2/components/index.yml (hunk 1 above)
Comment (huangalexpartoo) [reply rest_api/v2/components/index.yml:454]: I thought double was a bit overkill for longitude and latitude

File: rest_api/v2/components/index.yml (hunk 2 above)
Comment (huangalexpartoo) [reply rest_api/v2/components/index.yml:609]: yeah my bad massive refacto gone wrong 😩

**Pull request merged**